src/sentry/tasks/beacon.py

```diff
@@ -138,6 +140,10 @@ def send_beacon():
             "replays.24h": event_categories_count["replay"],
             "profiles.24h": event_categories_count["profile"],
             "monitors.24h": event_categories_count["monitor"],
+            "cpu_cores_available": psutil.cpu_count(),
+            "cpu_percentage_utilized": psutil.cpu_percent(),
+            "ram_available_gb": psutil.virtual_memory().total * byte_to_gigabyte,
+            "ram_percentage_utilized": psutil.virtual_memory().percent,
```

Comment:
is there a process to get updated consent from users to collect new data? without that I'm worried about the data collection backlash from this

Reply:
Good point, I've added a new consent form to collect this data here
https://github.com/getsentry/sentry/pull/65362